_style_guide/components/side-section.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Side Section
+type: components
+class: wide
+---
+
+<section class="side-section">
+  <h2>Aside 1</h2>
+  <aside class="info">
+    <p>The Core Team manages the development of Crystal and its ecosystem.</p>
+  </aside>
+
+  <div style="background-color: #ddd; min-width: 35ch;">
+    BODY 1
+  </div>
+</section>
+
+<section class="side-section reversed">
```

Comment:
What is the purpose of this reversed style?

Reply:
On smaller viewports the sidebar is positioned *after* the table.

Reply:
ah of course

Reply:
I _love_ the dynamics you're introducing 👏